ee_tests/oc-get-project-logs.sh

```diff
@@ -41,6 +41,14 @@ echo ---------- List all resources ----------------------
 echo "oc get all"
 oc get all
 
+echo ---------- List pvc ----------------------
```

Comment:
it's already on lines 61-63, one of these two should be deleted

Reply:
OK